.config/dotnet-tools.json

```diff
@@ -9,10 +9,10 @@
       ],
       "rollForward": false
     },
-    "gitversion.tool": {
```

Comment:
dont we need `gitversion` anymore?

Reply:
Nope, we don't use it at all.  At best it was a convenance to get the current tag but that's little of what we need.